cmd/config/aws/create_role_cmd.go

```diff
@@ -11,14 +11,28 @@ import (
 	"github.com/spf13/cobra"
 	log "go.uber.org/zap"
 	"io"
+	"time"
+)
+
+var (
+	useDevTemplate bool
 )
 
 const (
-	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
-	//TODO For usage documentation, visit...
+	configCreateRoleShort   = "AWS Create Role via CloudFormation Stack"
 	configCreateRoleLong    = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n"
 	configCreateRoleExample = "armory config aws create-role"
-	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://s3.us-west-2.amazonaws.com/static.cloud.armory.io/templates/aws/cloudformation/%siam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack-%s&param_AccountId=%s&param_ExternalId=%s&param_ArmoryRoleName=%s"
+	devTemplatePrefix       = "dev-"
+	armoryRoleName          = "ArmoryRole"
+
+	installPromptIntro  = "In order to deploy AWS resources, we need to create a Trust Relationship in your AWS account by adding a role that Armory can assume to execute deployments on your behalf."
```

Comment:
```suggestion
	installPromptIntro  = "In order to deploy AWS resources, Armory needs to create a Trust Relationship in your AWS account by adding an IAM role that Armory can assume to execute deployments on your behalf."
```

replace "we" with Armory